sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -420,17 +420,16 @@ export class AssessmentController {
           this.studentAssessmentService.saveAssessmentCalculationStartDate(
             assessmentId,
           );
-        const programYearTotals =
+        const workflowOutputsTotals =
           await this.assessmentSequentialProcessingService.getProgramYearTotals(
             assessmentId,
             {
               alternativeReferenceDate: new Date(),
             },
           );
-        // Updates the calculation start date and get the program year totals and for
-        // full-time the contribution program year totals in parallel.
-        const [programYearAwardsContributionTotals] = await Promise.all([
-          programYearTotals,
+        // Updates the calculation start date and get the program year totals and workflow outputs program year totals in parallel.
+        const [programYearTotals] = await Promise.all([
+          workflowOutputsTotals,
```

Comment:
Not part of this PR but line 424 has an unnecessary `await` that is making this `Promise.all` lose its goal.
Can we please remove the `await` at line 424?